src/golang/cmd/server/handler/edit_dynamic_engine.go

```diff
@@ -57,8 +59,10 @@ func (*EditDynamicEngineHandler) Headers() []string {
 
 const (
 	createAction      string = "create"
+	updateAction      string = "update"
 	deleteAction      string = "delete"
 	forceDeleteAction string = "force-delete"
+	configDeltaKey    string = "config_delta"
```

Comment:
Would appreciate some documentation for `configDeltaKey`. Where do we expect to use this key?